src/apps/immich/readme.md

```diff
@@ -0,0 +1,165 @@
+---
+# author: giga
+category: [Privacy]
+icon: device-camera-video 
+---
+
+## Prerequisites
+
+[!ref icon="check-circle" text="Complete First Steps"](/first-steps/1-vps-setup.md)
+
+## What is Immich
+
+Immich is a self-hosted, open-source platform for managing and sharing your personal media library. It provides a simple and intuitive interface for organizing and streaming your movies, TV shows, and music.
+
+## Installing Immich
+
+[!ref Official Docs](https://github.com/immich-app/immich)
+
+To start, create a new directory for the Immich container files (e.g. `mkdir /home/docker/immich`), start editing with `nano compose.yml` and paste the following contents:
+
+```yaml
+version: "3.8"
+
+services:
+  immich-server:
+    container_name: immich_server
+    image: ghcr.io/immich-app/immich-server:${IMMICH_VERSION:-release}
+    command: [ "start.sh", "immich" ]
+    volumes:
+      - ${UPLOAD_LOCATION}:/usr/src/app/upload
+    env_file:
+      - .env
+    depends_on:
+      - redis
+      - database
+      - typesense
+    restart: always
+
+  immich-microservices:
+    container_name: immich_microservices
+    image: ghcr.io/immich-app/immich-server:${IMMICH_VERSION:-release}
+    # extends:
+    #   file: hwaccel.yml
+    #   service: hwaccel
+    command: [ "start.sh", "microservices" ]
+    volumes:
+      - ${UPLOAD_LOCATION}:/usr/src/app/upload
+    env_file:
+      - .env
+    depends_on:
+      - redis
+      - database
+      - typesense
+    restart: always
+
+  immich-machine-learning:
+    container_name: immich_machine_learning
+    image: ghcr.io/immich-app/immich-machine-learning:${IMMICH_VERSION:-release}
+    volumes:
+      - model-cache:/cache
+    env_file:
+      - .env
+    restart: always
+
+  immich-web:
+    container_name: immich_web
+    image: ghcr.io/immich-app/immich-web:${IMMICH_VERSION:-release}
+    env_file:
+      - .env
+    restart: always
+
+  typesense:
+    container_name: immich_typesense
+    image: typesense/typesense:0.24.1@sha256:9bcff2b829f12074426ca044b56160ca9d777a0c488303469143dd9f8259d4dd
+    environment:
+      - TYPESENSE_API_KEY=${TYPESENSE_API_KEY}
```

Comment:
What's this about? Should this key be acquired from somewhere? Specification might be necessary. This env variable is also not present in the .env file created later.